### PR TITLE
[v1.2][ISSUE-306] Add comments to pii_removal,sentence_spit and document_extractor

### PR DESCRIPTION
**Pull request by chaojun-zhang**

### CODE REVIEW

File: RecDP/pyrecdp/primitives/llmutils/pii_remove.py [hunk 1]
```diff
@@ -4,6 +4,25 @@
...
+    """
+            Removes PII information from a DataFrame.
+
+            Args:
+                dataset: The DataFrame to be processed.
+                model_root_path: The path to the PII information removal model.
+                text_column: The name of the text column to be processed.
+                show_secret_column: Whether to add a new column to the DataFrame containing the masked PII information.
+                inplace: Whether to update the original text column.
+                entity_types: The types of PII information to be removed.
```
Comment (xuechendi) [inline RecDP/pyrecdp/primitives/llmutils/pii_remove.py:16]: please provide user a list of options for this parameter

File: RecDP/pyrecdp/primitives/operations/text_pii_remove.py [hunk 2]
```diff
@@ -48,6 +48,28 @@ def process(sample):
...
+            text_key(str): The name of the text field to be processed. Default is 'text'.
+            inplace(bool): Whether to update on the original text field. Default is True.
+            model_root_path(str): The root path to the model files.
+                For example, if you manually download the Hugging Face model `bigcode/starpii` for NER detection
+                from [Hugging Face](https://huggingface.co/bigcode/starpii)
+                to `/path/to/your_model_root_path/bigcode/starpii`,
+                then the `model_root_path` should be `"/path/to/your_model_root_path/"`.
+                Default is None, and it will auto download the huggingface model.
+            debug_mode(bool): Whether to enable debug mode. Default is False.
+            entity_types: The types of personal information to be removal. Default is None.
```
Comment (xuechendi) [inline RecDP/pyrecdp/primitives/operations/text_pii_remove.py:62]: please provide user a list of options for this parameter

**Review by xuechendi — CHANGES_REQUESTED** [review]
please fix the comments

File: RecDP/pyrecdp/primitives/operations/text_pii_remove.py (hunk 2 above)
Comment (chaojun-zhang) [reply RecDP/pyrecdp/primitives/operations/text_pii_remove.py:62]: updated

**Comment by chaojun-zhang** [comment]
> please fix the comments

Updated

**Pull request closed without merging**